### ISSUE TITLE
requiring elmi-to-json is slow on Windows

### ISSUE THREAD

**Issue by showell** [rtfeldman/node-test-runner#385]
Even if I just run `elm-test --help`, it take 3.5s on my relatively powerful Windows laptop to run that in a git bash shell.

I am not sure what I'm doing wrong, or if elm-test is just heavy like that.

If there isn't any easy way to speed this up, could it at least be documented as a known issue in the readme?

**Comment by showell** [comment]
This line is particularly slow (930ms on a fast, healthy Windows laptop with a current version of node):

```
var readElmiPath = require('elmi-to-json').paths['elmi-to-json'];
```

**Comment by Razzeee** [comment]
Updating `elmi-to-json` to the latest version 1.2.0 should presumably help with this https://github.com/stoeffel/elmi-to-json/releases

But I can't get the tests to pass after that.

**Comment by Razzeee** [comment]
Wos done in https://github.com/rtfeldman/node-test-runner/commit/7ac6505cf78aab0caad4d885a14a05c84eae1133 and should be in the next release

**Comment by rtfeldman** [comment]
@showell are you still seeing this in the current release, or can we close this?

**Comment by showell** [comment]
My issues were never resolved.  There's a thread here: https://elmlang.slack.com/archives/C0CLGCMMF/p1573296606141400

We figured out that it was Windows-specific issues.

**Comment by lydell** [comment]
@showell Would you mind trying out https://github.com/rtfeldman/node-test-runner/pull/442 and see if it’s faster for you?

1. Clone and checkout my branch
2. Run the Windows equivalent of `time ~/path/to/node-test-runner/bin/elm-test` in your project

**Comment by showell** [comment]
@lydell Sorry I can't help--I haven't been active in Elm lately, due to my day job.

**Comment by harrysarson** [comment]
I am happy to close this now. We no longer require elmi-to-json (on any OS which includes windows) so it cannot be slow. If anyone has any perf issues with the new approach (not yet released, will be version `0.19.1-revison5`) please open a new issue :)
